packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -161,6 +166,7 @@ const buttonStyles: ComponentSlotStylesPrepared<ButtonProps, ButtonVariables> =
         cursor: 'default',
         color: v.colorDisabled,
         boxShadow: 'none',
+        pointerEvents: 'none',
```

Comment:
Is this required by design? Or it was added because of the animation? Just checking in case if we have some unwanted behavior.

Reply:
Without this, the button will show the animation when the button is disabled. This property just makes the button unclickable when disabled.